packages/react-mutation-mapper/src/component/oncokb/ReferenceList.tsx

```diff
@@ -2,11 +2,11 @@ import { ICache } from 'cbioportal-frontend-commons';
 import * as React from 'react';
 import * as _ from 'lodash';
 
-import { ArticleAbstract } from '../../model/OncoKb';
 import ArticleAbstractItem from './ArticleAbstractItem';
 import PmidItem from './PmidItem';
 
 import styles from './listGroupItem.module.scss';
+import { ArticleAbstract } from 'cbioportal-frontend-commons/src/api/generated/OncoKbAPI';
```

Comment:
```suggestion
import { ArticleAbstract } from 'cbioportal-frontend-commons/';
```

also, can we move this into the top group of imports?